content/issues/2022-07-01-team-on-vacation.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Summer is here ☀️'
+date: '2022-07-04T08:00:00+00:00'
+resolved: false
```

Comment:
I am not not sure if we can't mark this "notice" as resolved, but having it unresolved will probably show that more prominently on the page... We'll see.

Reply:
I tried that, if I resolve it than it is displayed just in the history. Thus I marked it as not resolved...

Reply:
Thanks for checking that!